### PR TITLE
Fix consul

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
### Issues

Fixes https://github.com/wso2/product-microgateway/issues/2546

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: adapter/internal/oasparser/model/mgw_swagger.go [hunk 1]
```diff
@@ -1138,74 +1138,110 @@ func (swagger *MgwSwagger) PopulateSwaggerFromAPIYaml(apiData APIYaml, apiType s
...
 		var endpoints []Endpoint
 		endpointType := LoadBalance
-		for _, endpointConfig := range endpointConfig.SandBoxEndpoints {
-			sandBoxEndpoint, err := getHostandBasepathandPortWebSocket(endpointConfig.Endpoint)
-			if err == nil {
-				endpoints = append(endpoints, *sandBoxEndpoint)
+		var unProcessedURLs []interface{}
+		for _, endpointConfig := range endpointConfig.ProductionEndpoints {
+			if apiType == WS {
+				prodEndpoint, err := getHostandBasepathandPortWebSocket(endpointConfig.Endpoint)
```
Comment (praminda) [inline adapter/internal/oasparser/model/mgw_swagger.go:1148]: Shall we rename this function?
```suggestion
				prodEndpoint, err := getEndpointForWebsocketURL(endpointConfig.Endpoint)
```

Comment (AmaliMatharaarachchi) [reply adapter/internal/oasparser/model/mgw_swagger.go]: added https://github.com/wso2/product-microgateway/pull/2545/commits/038c7debac61e02744c4247a9e44e33bd4fe8a1b

**Review by praminda — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20211215.32

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211215.43

**Pull request merged**